### PR TITLE
BD-783 Request To Scan Metrological Github OCDM-Playready-Nexus

### PR DESCRIPTION
**Pull request by lakshmipriyap**
Incorporated header and LICENSE file changes to bring OCDM-Playready-Nexus repo to CMF

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2019 Metrological
+   Copyright [yyyy] [name of copyright owner]
```
Comment (wouterlucas) [inline LICENSE]: Same here, suppose to have a name of the company and year.

File: NOTICE [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+THE NOTICE FILE GIVES CREDIT TO AUTHORS AND INDICATES WHAT LICENSE APPLIES.
+OTHER FORMS ARE POSSIBLE DEPENDING ON INDIVIDUAL CASES.
+
+
+
+Copyright 2020 RDK Management
+Licensed under the Apache License, Version 2.0 
+
+Copyright 2016-2017 TATA ELXSI
+Copyright 2016-2017 Metrological
```
Comment (wouterlucas) [inline NOTICE]: we should update the dates to 2020

Comment (mhughesacn) [reply NOTICE]: These are the original header dates on the file.  I do not see why they have to change.  (Also, if Tata did not do this work for Metrological, then we do not have the right to change the copyright.)

File: NOTICE [hunk 3]
```diff
@@ -0,0 +1,13 @@
+THIS HEADER IS NOT A PART OF THE DEPLOYED NOTICE FILE.  PLEASE DELETE THE HEADER BEFORE COMMITTING.
+THE NOTICE FILE GIVES CREDIT TO AUTHORS AND INDICATES WHAT LICENSE APPLIES.
+OTHER FORMS ARE POSSIBLE DEPENDING ON INDIVIDUAL CASES.
+
+
+
```
Comment (mhughesacn) [inline NOTICE]: Please remove the first 6 lines.

**Pull request merged**